FilmsApp/View/ListController/ListViewController.swift

```diff
@@ -26,16 +27,40 @@ class ListViewController: UIViewController {
         fatalError("init(coder:) has not been implemented")
     }
     
+    func tableView(_ tableView: UITableView, trailingSwipeActionsConfigurationForRowAt indexPath: IndexPath) -> UISwipeActionsConfiguration? {
+        let myDel = UIContextualAction(style: .normal, title: "Favourite") { (_, _, complitionHand) in
+            DispatchQueue.global().async {
+            let film = Film_Data(context: self.context)
+            if let data = try? Data(contentsOf: (self.data.films[indexPath.row].imageURL)!) {
+                film.filmImage = data
+            }
+            film.filmTitle = self.data.films[indexPath.row].title
+            CoreDataManager.instance.saveContext()
+                DispatchQueue.main.async {
+                    complitionHand(true)
+                }
+            
+            }
+        }
+        myDel.image = UIImage(systemName: "star")
+        return UISwipeActionsConfiguration(actions:[myDel])
+    }
+    
     private func createTableView() {
-        myTableView = UITableView(frame: view.bounds, style: .plain)
-        view.addSubview(myTableView)
+        listTableView.translatesAutoresizingMaskIntoConstraints = false
+        self.view.addSubview(listTableView)
+        listTableView.topAnchor.constraint(equalTo: view.topAnchor, constant: 0).isActive = true
+        listTableView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 0).isActive = true
+        listTableView.trailingAnchor.constraint(equalTo: view.trailingAnchor).isActive = true
+        listTableView.bottomAnchor.constraint(equalTo: view.bottomAnchor).isActive = true
+        
```

Comment:
Consider using "NSLayoutConstraint.activate" method for activating constraints instead of setting "isActive" property